lib/system/selinux/bootstrap.go

```diff
@@ -124,8 +124,8 @@ func (r *BootstrapConfig) checkAndSetDefaults() error {
 		r.Path = utils.Exe.WorkingDir
 	}
 	if r.OS == nil {
-		metadata, err := monitoring.GetOSRelease()
-		if err != nil {
+		metadata, err := monitoring.GetOSRelease() //nolint:staticcheck,nolintlint
+		if err != nil {                            //nolint:staticcheck,nolintlint
```

Comment:
Because I have not figured out how to make this work for cases when linting with a different set of build tags - e.g. when some files are linux-only, this will trigger the `nolintlint` to emit a false positive about the linter being unused as the underlying implementation changes. I'm still looking for a better solution but I'll leave a comment about it.